setup.cfg

```diff
@@ -23,6 +23,7 @@ install_requires =
     ray>=2.0.1
     tabulate>=0.9.0
     networkx
+    bitsandbytes
```

Comment:
Let's add this as an additional extra require under a tag like `bnb`, so that users could install via `pip install trlx[bnb]`.